src/ResponseEmitter.php

```diff
@@ -0,0 +1,121 @@
+<?php
+namespace GuzzleHttp\Psr7;
+
+use Psr\Http\Message\ResponseInterface;
+
+/**
+ * Response emitter
+ */
+class ResponseEmitter
+{
+    /**
+     * Emits a response.
+     *
+     * @param ResponseInterface $response A response to emit.
+     * @param int $maxBufferSize Maximum output buffering size (pass 0 to emit all content at once).
+     * @param bool $normalizeHeaders Normalize header names while emitting. For example: Content-type -> Content-Type.
```

Comment:
Checking other files comments should be alinged.. same for other functions that exist here.

Reply:
Yes, it looks like this is a standard here. This is a matter of dispute, strictly speaking. For example, here: https://github.com/guzzle/psr7/blob/master/src/Uri.php#L206-L209 and here: https://github.com/guzzle/psr7/blob/master/src/Uri.php#L247-L250 This is a code, not a PHPDoc, but the logic is the same. Somebody love to align all `===`, etc. When renaming any of variables or function names we need to realign everything every time. If we need to align, I'll do this for sure.

P.S. https://github.com/guzzle/psr7/blob/master/src/PumpStream.php#L34-L41 This block needs fix :)

Reply:
Yes.. guzzle repos do not have codestyles applied through their CI.. which makes it easier to accept different codestyles... 

I have created a [PR](https://github.com/guzzle/guzzle/pull/2194) for codestyle but it is [pending](https://github.com/guzzle/guzzle/pull/2167#issuecomment-438008642)